<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_adresse_fournisseur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adresse</string>
   </property>
  </widget>
  <widget class="QLabel" name="titre_fournisseurs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Modification des informations du fournisseur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type_marchandise">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>255</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type de marchandise</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_nom_fournisseur">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_type_marchandise">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_email_fournisseur">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_adresse_fournisseur">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_annuler_fournisseur">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>328</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color: rgb(255, 255, 255);
		background-color: rgb(35, 35, 35);
		border: 0px solid;
		border-radius:5px
}

QPushButton:hover {
		background-color: rgb(85, 170, 255);
}</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_num_fournisseur">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_fournisseur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_fournisseur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>135</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N° téléphone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_enregistrer_fournisseur">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>328</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color: rgb(255, 255, 255);
		background-color: rgb(35, 35, 35);
		border: 0px solid;
		border-radius:5px
}

QPushButton:hover {
		background-color: rgb(85, 170, 255);
}</string>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom_fournisseur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom du fournisseur</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
